src/webprojects/fxa.md

```diff
@@ -0,0 +1,115 @@
+# Firefox Accounts (FxA): how to localize and test
+
+Firefox Accounts (or FxA) refers to https://accounts.firefox.com/signin. There are three components in Pontoon that make up [FxA](https://pontoon.mozilla.org/projects/firefox-accounts/):
+
+* `LC_MESSAGES/client.po`: the content server for UI and frontend.
+* `LC_MESSAGES/server.po`: strings for emails and backend templates.
+* `Main.ftl`: payments for both frontend and backend.
+
+The above could change over time as the team is planning to migrate .po files to .ftl, which may involve restructuring the files.
+
+## Key links
+
+* Production: https://accounts.firefox.com/
+* Staging: https://latest.dev.lcip.org/
+* Repository: https://github.com/mozilla/fxa-content-server-l10n
+* Pontoon: https://pontoon.mozilla.org/projects/firefox-accounts/
+
+## Localizing the strings
+
+FxA lives in two file formats. The shorter a string is, especially the kind containing variable or placeable, the more context you need to make sure you get the translation right. Look for clues or ask others for clarification:
+
+* String comments: Pay attention to the comment or string ID. In Pontoon, the comments are displayed above the area where you can enter your translation for a string. Some strings are used in more than one place. Finding the appropriate translation that works broadly is important.
+* [Matrix channel](https://chat.mozilla.org/#/room/#l10n-community:mozilla.org): ping the L10n PM in charge with any issues you may have.
+* [Web projects mailing list](https://groups.google.com/g/mozilla.dev.l10n.web): Raise questions here so others may benefit from the information.
+
+## Errors in translations can break the build for all languages
+
+During translation, do not alter codes in a string, such as variables and HTML markup.
+
+For .po file format, a few FxA contacts would be notified when a code in localized string is broken. Any subsequent commits by other localizers in other languages will trigger this warning until the broken string is fixed by one of the people notified. Here are a few common errors seen in localized content:
+
+✅ `The recipient’s consent is required. <a %(escapedLearnMoreAttributes)s>Learn more</a>`<br>
```

Comment:
You still have `<br>`after the code. Why? They're not needed in markdown, just create a new line with enter. If you want a paragraph, leave an empty lines.